scylla/src/transport/session.rs

```diff
@@ -357,6 +401,30 @@ impl Session {
             .ok_or_else(|| anyhow!("fatal error, broken invariant: no connections available"))?
             .any_connection()?)
     }
+
+    pub fn latency_percentile(&self, percentile: f64) -> u64 {
```

Comment:
Let's remove these helper functions and instead allow the user to access `&Metrics` via `get_metrics()` or something similar. That way, the interface for metrics is contained in another class and doesn't bloat the `Session` interface.